specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json

```diff
@@ -99,7 +99,7 @@
         "summary": "Get analysis status and results",
         "parameters": [
           {
-            "$ref": "#/parameters/JobId"
+            "$ref": "#/parameters/AnalyzeJobId"
```

Comment:
is that because of the `uuid` issue?

Reply:
Yes, I decided to add a new value for the job ID for Analyze because Guid has already been implemented for the Healthcare service and I didn't want to break anything.  The service team who owns the Analyze frontend has told us that they won't have time to change their job ID to a Guid in time for the preview release.